### ISSUE TITLE
Invoking with script invokes with wrong message

### ISSUE THREAD

**Issue by juhakivekas** [zaproxy/zaproxy#4085]
The "Invoke with script..." option of history table entries does not work on the selected message, but the newest message which is represented by same node.

```
function invokeWith(msg) {
	print(msg.getRequestBody().length());
}
```

1. `curl --proxy https://127.0.0.1:8080 https://asdf.com --data '12345678'`
2. Invoke the message with the above targeted script, the scripts prints 8
3. `curl --proxy https://127.0.0.1:8080 https://asdf.com --data '1234'`
4. Invoke the first message with the script again, the script prints 4

The expected behavior<sup>[irc]</sup> is for the script to print the actual length of the first message on both runs.

**Comment by thc202** [comment]
Fixed in zaproxy/zap-extensions#1171.

**Comment by thc202** [comment]
Released in version 23 of Script Console add-on.
